### PR TITLE
Fix Forem Ownership Field

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
I mistakenly renamed a field from `forem_owner_secret` to `owner_secret`. This causes a failure when its expecting `forem_owner_secret` in the Registrations controller. 

This has no impact in production since. this view was behind a feature flag. 

## Related Tickets & Documents
https://github.com/forem/forem/issues/14647

## QA Instructions, Screenshots, Recordings
- Delete the database with `rails db:drop; rails db:create; rails db:migrate; rails data_updates:run`. 
- Enale the feature flag by. running `FeatureFlag.enable(:creator_onboarding)`
- Thereafter, go to the signup page, and try signing up.
- You signup should be successful. 

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: https://github.com/orgs/forem/projects/36#card-68001436, I'll tackle this before we turn the feature flag on. 
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be updated. I have filled out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can help update the Admin Docs appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by ludwiczakpawel — APPROVED** [review]

**Review by aitchiss — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (citizen428) [inline .github/PULL_REQUEST_TEMPLATE.md:61]: This change probably shouldn't be part of this PR.

File: app/views/shared/authentication/_forem_creator_signup.html.erb
Comment (citizen428) [inline app/views/shared/authentication/_forem_creator_signup.html.erb:87]: Could it make sense to wrap this up in our `ForemInstance` model? Something like `ForemInsance.needs_owner_secret?` or something?

Comment (Ridhwana) [reply app/views/shared/authentication/_forem_creator_signup.html.erb:87]: This is a good idea - thanks Michael.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (Ridhwana) [reply .github/PULL_REQUEST_TEMPLATE.md:61]: Fixed, now the update is simply formatting issues that the commit hooks updated:)

**Comment by Ridhwana** [comment]
@citizen428 I've made these updates and l'll merge this once you've approved :)

File: app/javascript/chat/chat.jsx
Comment (Ridhwana) [inline app/javascript/chat/chat.jsx:1497]: Some formatting commit hook changes.

**Review by nickytonline — APPROVED** [review]
🚀🚢⛴🛳

**Review by citizen428 — APPROVED** [review]

**Pull request merged**
